src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+//new class Excavator
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message should indicate what type of `Machine` is working right now. Consider making the message more informative, e.g., 'Excavator is working.'

Reply:
Thank you for comments. I made new commit